docs/operator-manual/high_availability.md

```diff
@@ -64,6 +64,8 @@ controller replicas. To enable sharding increase the number of replicas in `argo
 and repeat the number of replicas in the `ARGOCD_CONTROLLER_REPLICAS` environment variable. The strategic merge patch below
 demonstrates changes required to configure two controller replicas.
 
+* By default, the controller will update the cluster information every 10 seconds,if there is a problem with your cluster network environment that is causing the update time to take a long time, you can try modifying the environment variables `ARGO_CD_UPDATE_CLUSTER_INFO_TIMEOUT` to increase the timeout time, the unit is seconds.
```

Comment:
```suggestion
* By default, the controller will update the cluster information every 10 seconds. If there is a problem with your cluster network environment that is causing the update time to take a long time, you can try modifying the environment variable `ARGO_CD_UPDATE_CLUSTER_INFO_TIMEOUT` to increase the timeout (the unit is seconds).
```

Reply:
thanks for your review, I have solved it. PLAT